src/StreetNameRegistry.Consumer.Read.Postal/PostalConsumerItem.cs

```diff
@@ -0,0 +1,75 @@
+namespace StreetNameRegistry.Consumer.Read.Postal
+{
+    using System;
+    using Microsoft.EntityFrameworkCore;
+    using Microsoft.EntityFrameworkCore.Metadata.Builders;
+    using StreetNameRegistry.Infrastructure;
+
+    public class PostalConsumerItem
+    {
+        public string PostalCode { get; set; }
+        public string? NisCode { get; set; }
+        public PostalStatus Status { get; set; }
+        public string? NameDutch { get; set; }
+        public string? NameFrench { get; set; }
+        public string? NameGerman { get; set; }
+        public string? NameEnglish { get; set; }
```

Comment:
What with multiple names?
https://api.prd.basisregisters.vlaanderen.be/v1/postinfo/8200